lblod/frontend-dashboard
# [DL-5958] Allow admin users to impersonate other accounts

Users with the correct role will see a new dropdown menu in the top which can be used to impersonate other accounts.

This feature is only enabled when the `EMBER_ADMIN_ROLE` environment variable is set.

To test:

1. proxy to Loket QA, or your local loket environment
2. edit the config/environment.js file and set `adminRole` to `LoketLB-admin`
3. Log in using the the "Agentschap binnenlands bestuur" mock user
4. Test the impersonation feature by comparing it with the data of other mock users. (The data should be the same as if you logged in as the actual mock user).

Builds on #35 so needs to be merged after that one.

Based on the code from this PR: https://github.com/lblod/frontend-worship-decisions/pull/22
